client/src/components/Item/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+
+import {
+  ItemContainer,
+  ItemDiv,
+  ItemImage,
+  Line,
+  ItemTitle,
+  ItemText,
+  ItemMainData,
+  ItemSubtitle,
+  ItemTag,
+  Column,
+  ItemTags,
+} from './item.style';
+
+class GetDetails extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <React.Fragment>
```

Comment:
You don't need this extra React Fragment as you're using a div.

Reply:
removed